### PR TITLE
Updated composer dependencies

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR updates composer.json to upgrade to symfony3.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,25 +7,24 @@
...
             "name": "Sulu Community",
-            "homepage": "https://github.com/sulu/sulu/contributors"
+            "homepage": "https://github.com/sulu/SuluThemeBundle/contributors"
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
-        "liip/theme-bundle": "1.4.*"
+        "php": "~5.5 || ~7.0",
+        "liip/theme-bundle": "^1.4"
```
Comment (danrot) [inline composer.json]: Shouldn't that also be `~1.4`?

**Review by danrot — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (wachterjohannes) [reply composer.json]: right

Comment (wachterjohannes) [reply composer.json]: @danrot fixed

**Comment by wachterjohannes** [comment]
@danrot there is currently a bug in the theme-bundle with 3.2.1 https://github.com/liip/LiipThemeBundle/pull/159/files i would say we will wait for this fix.

**Comment by wachterjohannes** [comment]
here the link to the issue https://github.com/liip/LiipThemeBundle/issues/157

**Comment by wachterjohannes** [comment]
can be merged now (: https://github.com/liip/LiipThemeBundle/releases/tag/1.4.3

**Review by danrot — APPROVED** [review]

**Pull request merged**